native/src/main/java/io/ballerina/lib/ibm.ibmmq/QueueManager.java

```diff
@@ -64,6 +67,21 @@ public static Object init(BObject queueManager, BMap<BString, Object> configurat
         return null;
     }
 
+    public static Object accessTopic(Environment env, BObject queueManagerObject, BString topicName,
```

Comment:
I think we are not using the `Environment` in our logic.
```suggestion
    public static Object accessTopic(BObject queueManagerObject, BString topicName,
```